.github/cspell.json

```diff
@@ -1,8 +1,9 @@
-// cspell Settings (https://github.com/streetsidesoftware/cspell/tree/main/packages/cspell#cspelljson)
 {
+"_comment1": "cspell Settings (https://github.com/streetsidesoftware/cspell/tree/main/packages/cspell#cspelljson)",
+
     "version": "0.2",
     "language": "en",
-    // list of words to always be considered correct (may need additions later)
+    "_comment2": "list of words to always be considered correct (may need additions later)",
```

Comment:
I'm not sure that this change is a good idea. This JSON file is in what is known as "JSONC", or JSON with comments; if the integration team would like to ensure this file is in JSON format, it would be better to remove these comments, as they may have unintended effects if they are added to the JSON structure.